src/c_kzg_4844.h

```diff
@@ -45,10 +45,12 @@ typedef blst_p2 g2_t;         /**< Internal G2 group element type */
 typedef blst_fr fr_t;         /**< Internal Fr field element type */
 
 typedef struct { uint8_t bytes[32]; } Bytes32;
-typedef struct { uint8_t bytes[BYTES_PER_COMMITMENT]; } KZGCommitment;
-typedef struct { uint8_t bytes[BYTES_PER_PROOF]; } KZGProof;
+typedef struct { uint8_t bytes[48]; } Bytes48;
 typedef struct { uint8_t bytes[BYTES_PER_BLOB]; } Blob;
 
+typedef Bytes48 KZGCommitment;
```

Comment:
I probably missed it, but are these used for something?

Reply:
They are used as inputs/outputs in internal functions as in the spec. They signify a KZG proof or commitment which has been deserialized and normalized.